lib/libutils/isoc/arch/arm/arm32_aeabi_divmod.c

```diff
@@ -103,7 +106,11 @@ unsigned __aeabi_uidiv(unsigned numerator, unsigned denominator)
 	return qr.q;
 }
 
+#ifndef __clang__
```

Comment:
This should be defined where `__no_stackprot` is defined instead to avoid having `#ifndef __clang__` all over the place.